mautic/mautic
# Removing index_dev.php

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [X]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [X] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation-new/pull/126
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR removes the `index_dev.php` and edits environment variables to detect if the environment is `dev`, `prod`, or `testing` and render the assets accordingly. also it define if `debugging` should be enabled or not.




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You should see in gitpod that we are not using `index_dev.php` but `index.php`
3. You can see debugging features if you are in git pod
